### DISCUSSION TITLE
Cannot use @can or auth()->user()->can() function

### DISCUSSION THREAD

**Discussion by charlottedrb** [spatie/laravel-permission#1971] [Q&A]
Hi, 
I'm trying to set up a permission and role management on a Laravel 8 / php 8 projet with Spatie. 
I'm having an issue with the "can" directive in Blade and in Laravel. 
The "can" always return "true" so there isn't any control on the user's permissions. 

The only solution I had now, is to use the hasPermissionTo method and to overwrite the natives Middleware of Spatie package.

If anyone already had this type of problem, thanks in advance for your help !

**Comment by erikn69** [comment]
Duplicate of #1955, The gate is tested, and i can't replicate this behaivor, 
https://github.com/spatie/laravel-permission/blob/08cc9bdbf89640cb1d64fb01eb9513c1cbb2549d/tests/GateTest.php#L40-L53

It would be very helpful if someone make a minimal laravel application on fresh install, where problem is demonstrated or at least a failing test, example: [IssueTest](https://github.com/Onekone/spatie-issue-example/blob/main/tests/Feature/IssueTest.php)

**Comment by mohammadnazari110** [comment]
https://github.com/spatie/laravel-permission/issues/1983
